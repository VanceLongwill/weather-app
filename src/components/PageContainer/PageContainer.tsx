import * as React from 'react'
import styled from 'styled-components'

const Wrapper = styled.section`
  padding: 4em;
  background: #fefefe;
  margin: 0 auto;
`

interface IPageContainerProps {
  children?: React.ReactNode
}

export function PageContainer(props: IPageContainerProps) {
  return <Wrapper>{props.children}</Wrapper>
}
