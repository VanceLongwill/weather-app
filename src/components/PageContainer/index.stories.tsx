import { storiesOf } from '@storybook/react'
import * as React from 'react'
import PageContainer from './'

storiesOf('PageContainer', module).add('with no children', () => {
  return <PageContainer />
})
