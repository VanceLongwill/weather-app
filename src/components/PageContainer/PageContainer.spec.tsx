import { shallow } from 'enzyme'
import * as React from 'react'

import { PageContainer } from './PageContainer'

describe('PageContainer', () => {
  it('it should match snapshot', () => {
    const component = shallow(<PageContainer />)
    expect(component).toMatchSnapshot()
  })
})
