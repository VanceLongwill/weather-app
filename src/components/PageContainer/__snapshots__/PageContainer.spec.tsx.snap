// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PageContainer it should match snapshot 1`] = `<PageContainer__Wrapper />`;
